src/mapping.rs

```diff
@@ -1060,4 +1097,32 @@ androidx.activity.OnBackPressedCallback
             ],
         );
     }
+
+    #[test]
+    fn cleanup() {
+        let bytes = br#"
+# compiler: R8
+# common_typos_disable
+
+androidx.activity.OnBackPressedCallback->c.a.b:
+androidx.activity.OnBackPressedCallback -> c.a.b:
+# {"id":"sourceFile","fileName":"Test.kt"}
+    boolean mEnabled -> a
+  boolean mEnabled -> a
+    java.util.ArrayDeque mOnBackPressedCallbacks -> b
+    1:4:void onBackPressed():184:187 -> c
+androidx.activity.OnBackPressedCallback 
+-> c.a.b:
+        "#;
+
+        let mut out = Vec::new();
+
+        cleanup_mapping_file(&mut Cursor::new(bytes), &mut out).unwrap();
+
+        let expected = r#"androidx.activity.OnBackPressedCallback -> c.a.b:
+# {"id":"sourceFile","fileName":"Test.kt"}
+    1:4:void onBackPressed():184:187 -> c
+"#;
+        assert_eq!(std::str::from_utf8(&out).unwrap(), expected);
```

Comment:
Might as well compare with a `br#""#` raw byte string instead of using `from_utf8`.

Reply:
Right, I did that originally because the test failed and I couldn't read the results.